src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -76,6 +76,18 @@
                DisplayName="Annotations" />
   </EnumProperty>
 
+  <BoolProperty Name="Prefer32Bit"
+                DisplayName="Prefer 32-bit"
+                Description="Set the 32-bit preferred flag for the application. Only applies to executables."
```

Comment:
Also, it would be good if the `Description` explained a little of the effect this option has on the output. For example:

```suggestion
                Description="Run in 32-bit mode on systems that support both 32-bit and 64-bit applications."
```

Reply:
Updated.